src/main/java/org/broadinstitute/consent/http/service/DaaService.java

```diff
@@ -166,8 +166,8 @@ public void sendDaaRequestEmails(User user, Integer daaId) throws Exception {
         DataAccessAgreement daa = findById(daaId);
         String daaName = daa.getFile().getFileName();
         User toUser = new User();
-        toUser.setEmail(signingOfficial.email);
-        toUser.setDisplayName(signingOfficial.displayName);
+        toUser.setEmail(signingOfficial.getEmail());
```

Comment:
The `SimplifiedUser` changes in this PR are just some cleanup I did along the way.